delivery_transsmart/models/stock_picking.py

```diff
@@ -99,8 +99,17 @@ def get_invoice_name(self):
     def _transsmart_document_from_stock_picking(self):
         # TODO: Move to transsmart settings
         # HARDCODED weight correction: +3% +0.05kg , round to 1 decimal (=0.1kg)
-        weight = float(round(((self.weight * 1.03) + 0.05), 1))
-
+        #weight = float(round(((self.weight * 1.03) + 0.05), 1))
+        # the code below assumes that trassmart_id is unique...
+        carrier = self.env['res.partner'].search(
+                [('transsmart_id', '=', self.get_carrier_id())])
+        colli_information = carrier.transsmart_package_type_id
+        if not colli_information:
+            raise Warning(_(
+                "You have not set a package type for"
+                " the carrier {}, please do".format(carrier.name) + 
```

Comment:
Here you have a variable part in a translatable string. That is something to avoid.